tools/tidb-binlog-cluster.md

```diff
@@ -0,0 +1,566 @@
+# valid values are "mysql", "pb", "tidb", "flash", "kafka"
+db-type = "mysql"
+
+# the downstream mysql protocol database
+[syncer.to]
+host = "172.16.10.72"
+user = "root"
+password = "123456"
+port = 3306
+# Time and size limits for flash batch write
+# time-limit = "30s"
+# size-limit = "100000"
+```
+
+2. 以下游为 pb 为例
+
+```
+$ cd /home/tidb/tidb-ansible/conf
+$ cp drainer.toml drainer_pb_drainer.toml
+$ vi drainer_pb_drainer.toml
+```
+
+db-type 设置为 "pb"。
+
+```
+# downstream storage, equal to --dest-db-type
+# valid values are "mysql", "pb", "tidb", "flash", "kafka"
+db-type = "pb"
+
+# Uncomment this if you want to use pb or sql as db-type.
+# Compress compresses output file, like pb and sql file. Now it supports "gzip" algorithm only. 
+# Values can be "gzip". Leave it empty to disable compression. 
+[syncer.to]
+compression = ""
+dir = "data.drainer"
+```
+
+#### 部署 Drainer
+
+```
+$ ansible-playbook deploy_drainer.yml
+```
+
+#### 启动 Drainer
+
+```
+$ ansible-playbook start_drainer.yml
+```
+
+## 使用 Binary 部署 TiDB-Binlog 
+### 下载官方 Binary
+
+```
+TiDB（pump client）
+wget https://download.pingcap.org/tidb-v2.0.7-binlog-dev-linux-amd64.tar.gz
+wget https://download.pingcap.org/tidb-v2.0.7-binlog-dev-linux-amd64.sha256
+ 
+# 检查文件完整性，返回 ok 则正确
+sha256sum -c tidb-v2.0.7-binlog-dev-linux-amd64.sha256
+ 
+Pump && Drainer
+wget https://download.pingcap.org/tidb-binlog-new-linux-amd64.tar.gz
+wget https://download.pingcap.org/tidb-binlog-new-linux-amd64.sha256
+ 
+# 检查文件完整性，返回 ok 则正确
+sha256sum -c tidb-binlog-new-linux-amd64.sha256
+```
+
+### 使用样例
+假设我们有三个 PD，一个 TiDB，另外有两台机器用于部署 Pump，一台机器用于部署 Drainer。 各个节点信息如下：
+```
+TiDB="192.168.0.10"
+PD1="192.168.0.16"
+PD2="192.168.0.15"
+PD3="192.168.0.14"
+Pump="192.168.0.11"
+Pump="192.168.0.12"
+Drainer="192.168.0.13"
+```
+以此为例，说明 Pump/Drainer 的使用.
+
+#### 使用 binary 部署 Pump
+
+1. Pump 命令行参数说明（以在 “192.168.0.11” 上部署为例）
+
+    ```
+    Usage of Pump:
+    -L string
+        日志输出信息等级设置: debug, info, warn, error, fatal (默认 "info")
+    -V
+        打印版本信息
+    -addr string
+        Pump 提供服务的 RPC 地址(-addr="192.168.0.11:8250")
+    -advertise-addr string
+        Pump 对外提供服务的 RPC 地址(-advertise-addr="192.168.0.11:8250")
+    -config string
+        配置文件路径，如果你指定了配置文件，Pump 会首先读取配置文件的配置；
+        如果对应的配置在命令行参数里面也存在，Pump 就会使用命令行参数的配置来覆盖配置文件里面的。
+    -data-dir string
+        Pump 数据存储位置路径
+    -enable-tolerant
+        开启 tolerant 后，如果 binlog 写入失败，Pump 不会报错（默认开启）
+    -gc int
+        pump 只保留多少天以内的数据 (默认 7)
+    -heartbeat-interval int
+        Pump 向 PD 发送心跳间隔 (单位 秒)
+    -log-file string
+        log 文件路径
+    -log-rotate string
+        log 文件切换频率，hour/day
+    -metrics-addr string
+        Prometheus pushgateway 地址，不设置则禁止上报监控信息
+    -metrics-interval int
+        监控信息上报频率 (默认 15，单位 秒)
+    -pd-urls string
+        PD 集群节点的地址 (-pd-urls="http://192.168.0.16:2379,http://192.168.0.15:2379,http://192.168.0.14:2379")
+    ```
+
+2. Pump 配置文件（以在 “192.168.0.11” 上部署为例）
+    ```
```

Comment:
Add a blank line between line 295&296 to avoid possible display error on PingCAP website.